apiary.apib

```diff
@@ -250,14 +250,25 @@ or set account properties or destroy their accounts.
 
             Accept: application/vnd.api+json
 
+    + Attributes (object)
+        + data (object, required)
+            + type (string, required)
+            + attributes (object, required)
+                + username (string, required)
+                + password (string, required)
+                + createdAt (string, optional) - timestamp the user account was created on the client
```

Comment:
To show the difference between optional and required request body attributes, it was necessary to add this. What other optional attributes are can be sent? Also, id inside `attributes` or outside like it is now?